### PR TITLE
Bump c/Storage to 1.10

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

Bump to v1.10 for container/storage.

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
@vrothberg PTAL

**Comment by vrothberg** [comment]
LGTM :) Thanks, @TomSweeneyRedHat !

File: vendor.conf [hunk 1]
```diff
@@ -6,7 +6,7 @@ github.com/containernetworking/cni v0.7.0-alpha1
 github.com/containers/image 93bced01015eb94bec4821df1876314be8197680
 github.com/boltdb/bolt v1.3.1
 github.com/containers/libpod v1.0
-github.com/containers/storage 06b6c2e4cf254f5922a79da058c94ac2a65bb92f
+github.com/containers/storage 0a30cf16081e9a4c4bc0d2571c29c7f5201f06a5
```
Comment (rhatdan) [inline vendor.conf]: Can you pull via v1.10 instead of UUID?

Comment (TomSweeneyRedHat) [reply vendor.conf]: Apparently I can use v1.10 instead, just changed, repushing shortly.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by vrothberg** [comment]
bot, retest this please

**Comment by TomSweeneyRedHat** [comment]
I love homu....

**Comment by TomSweeneyRedHat** [comment]
@rhatdan homu isn't cooperating, worth asking the bot to try again?

**Comment by vrothberg** [comment]
> @rhatdan homu isn't cooperating, worth asking the bot to try again?

Could you repush to fire off everything?

**Comment by TomSweeneyRedHat** [comment]
@vrothberg I could, but was hoping @rhatdan had another thought.  If not, I'll repush and pray.

**Comment by TomSweeneyRedHat** [comment]
Happy Green Test Buttons after repushing.

**Comment by vrothberg** [comment]
@rh-atomic-bot r=@rhatdan

**Pull request closed without merging**